src/katsdpmodels/primary_beam.py

```diff
@@ -149,20 +211,20 @@ def frequency_resolution(self) -> u.Quantity:
         have diminishing returns compared to sampling at this resolution and
         interpolating.
         """
-        raise NotImplementedError()
+        raise NotImplementedError()      # pragma: nocover
 
     def inradius(self, frequency: u.Quantity) -> float:
         """Maximum distance from the pointing centre at which model has full coverage."""
-        raise NotImplementedError()
+        raise NotImplementedError()      # pragma: nocover
 
     def circumradius(self, frequency: u.Quantity) -> float:
         """Maximum distance from the pointing centre at which model has any coverage."""
-        raise NotImplementedError()
+        raise NotImplementedError()      # pragma: nocover
 
```

Comment:
is there a reason the return type for inradius and circumradius is not an astropy.units.Quantity?

Reply:
They're direction cosines, which are unitless. I've generally gone for plain float (or ndarray, when arrays) for unitless quantities to simplify use.